src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -17,6 +17,14 @@ namespace Microsoft.CodeAnalysis
     /// </summary>
     internal abstract class ErrorLogger
     {
+        private class NullErrorLogger : ErrorLogger
+        {
+            public override void LogDiagnostic(Diagnostic diagnostic)
+            {
+            }
+        }
```

Comment:
Consider moving to TestUtilities since this is only used by tests. See `DiagnosticBagErrorLogger` for similar class.

Reply:
Done.